### ISSUE TITLE
Doesn't support prefix

### ISSUE THREAD

**Issue by viennv1709** [windicss/windicss-intellisense#67]
The intellisense doesn't support  prefix. It should support class like this`tw-cursor-pointer` instead of `cursor-pointer`. This is my `tailwind.config.js` file.
```typescript
module.exports = {
  prefix: 'tw-',
  prefixer: true,
  darkMode: 'class',
}
```

**Comment by alexanderniebuhr** [comment]
/cc @antfu 
i would expect the processor to give back prefixed utilities, which seems not to be the case. or are we suposed to add prefix afterwards?
```ts
    const processor = new Processor(config);
    ...
    const staticUtilities = processor.resolveStaticUtilities(true);
```

**Comment by NicolaSpadari** [comment]
Yes, intellisense on windi with custom prefix would be very useful.
I found the official tailwindcss vscode extension does this: https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/packages/tailwindcss-language-service/src/completionProvider.ts#L310

when you define in vscode settings this config: `"tailwindCSS.experimental.classRegex": ["tw-`([^`]*)"]`

**Comment by voorjaar** [comment]
from v0.16.4 prefix should been supported
